most-run/src/index.ts

```diff
@@ -1,21 +1,50 @@
 import {Stream} from 'xstream';
 import * as most from 'most';
 import {Stream as MostStream} from 'most';
-import {setAdapt} from '@cycle/run/lib/adapt';
+import {setAdapt} from '../../run/src/adapt';
```

Comment:
I know you need that for testing, but you could use `pnpm link` instead.
This wont work for release and could be forgotten

Reply:
How could we make it work for CI for now?